modules/restoring-the-mysql-database.adoc

```diff
@@ -0,0 +1,147 @@
+:_mod-docs-content-type: PROCEDURE
+
+//included in backing-up-and-restoring-3scale-using-oadp.adoc assembly
+
+[id="restoring-the-mysql-database_{context}"]
+= Restoring the mysql database
+
+The restoration of the mysql database re-creates the Pod, ReplicationController, DeploymentConfig, additional PV, and persistent volume claim (PVC).
+
+NOTE: Do not delete the default PV and PVC associated with the database. 
+
+.Prerequisites
+
+* You have restored the secrets and APIManager custom resources.
+
+.Procedure
+
+. Scale down the threescale operator:
++
+[source,terminal]
+----
+$ oc scale deployment threescale-operator-controller-manager-v2 --replicas=0 -n threescale
+deployment.apps/threescale-operator-controller-manager-v2 scaled
+----
+
+. Scale down all the DeploymentConfigs:
++
+[source,terminal]
+----
+$ cat ./scaledowndc.sh
+
+for dc in apicast-production apicast-staging backend-cron backend-listener backend-redis backend-worker system-app system-memcache system-mysql system-redis system-searchd system-sidekiq zync zync-database zync-que; do
+    oc scale dc/$dc --replicas=0 -n threescale
+done
+----
++
```

Comment:
```suggestion
.. Run the script you created by using the following command:
```
Or something like this. It is not one step and one command; we're clearly doing two things here.